hscontrol/poll.go

```diff
@@ -30,9 +30,19 @@ type UpdateNode func()
 func logPollFunc(
 	mapRequest tailcfg.MapRequest,
 	node *types.Node,
-) (func(string), func(error, string)) {
+) (func(string), func(string), func(error, string)) {
 	return func(msg string) {
-			log.Info().
+			log.Trace().
```

Comment:
I've never worked with zerolog before, so I am not sure if this is the way to go

Reply:
Looks good 👍